### DISCUSSION TITLE
Questions related to Task 1

### DISCUSSION THREAD

**Discussion by tejuafonja** [AISaturdaysLagos/cohort_structure#10] [Q&A]
Please post any questions relating to [task 1](https://github.com/AISaturdaysLagos/cohort7_practicals/tree/main/Data%20Science%20Notebook%20Frameworks) here

**Comment by dnzengou** [comment]
@theYorubayesian , not sure this box dedicated to answers is the right one... but i give it a try:

i have a concern regarding the task for this week:
- I don't see a link between the housing data and the season function
- I don't see how to go from the example with the first season to define the function for the others. IMHO there are missing information... Unless i missed something.
Thanks for the help.

**Reply by israelarowolo** [reply]
- Well, apparently you don't need the housing data for the assignment, it's just to get you familiar with Google colab. 
- Also, you can choose whatever metric you want to define the season function, could be as simple as using month and date to determine the season, or something even more complex, anything you're fine with really.......I think.

**Reply by tejuafonja** [reply]
@dnzengou thanks for raising this. We've just updated the task. Here's the [link](https://github.com/AISaturdaysLagos/cohort7_practicals/blob/main/Data%20Science%20Notebook%20Frameworks/DataScienceNotebook.ipynb). However, if you've already solved it - that's also fine 😎. The purpose of the task is to get you up-to-speed with Git, Colab, Github, and collaborating :))

**Reply by dnzengou** [reply]
Thanks you two for the tips. Appreciate.

**Reply by dnzengou** [reply]
Quick question though: what does the threshold "1800" represent?

**Comment by raq-man11** [comment]
Hello,
Please I tried using the method utilized in the practical session today to download the IPYNB File from Github. I selected "All Files", but it downloads as a text file and I can't upload it to Colab. Please any help here?

**Reply by Timmy-id** [reply]
You can go to rename file and change the file extension from .txt to .ipynb
